src/reducers/app.js

```diff
@@ -55,13 +55,28 @@ function isUrlSetupDone(state: boolean = false, action: Action) {
       return state;
   }
 }
+
+function hasZoomedViaMousewheel(state: boolean = false, action: Action) {
+  switch (action.type) {
+    case 'HAS_ZOOMED_VIA_MOUSEWHEEL': {
+      return true;
+    }
+    default:
+      return state;
+  }
+}
+
```

Comment:
My reasoning is that I want to show this notice sparingly to the user, and I thought the preference should be shared between any panels that show it. The "Marker Chart" and the "Stack Chart" both use it. Once we have a marker chart reducer I'd be fine revisiting this. For now I think it's simpler to throw it in with the rest of the `app` logic since its shared.